shared/templates/audit_rules_privileged_commands/ansible.template

```diff
@@ -26,6 +26,24 @@
       - "{{ find_{{{ NAME }}}.files | map(attribute='path') | list | first }}"
   when: find_{{{ NAME }}}.matched is defined and find_{{{ NAME }}}.matched > 0
 
+{{% if product == "sle12" %}}
```

Comment:
This change is present in #6631 as well. I'll ask them to remove from there

Reply:
Change was pulled fro, 6331